packages/stark-core/src/modules/error-handler/error-handling/error-handler.ts

```diff
@@ -0,0 +1,48 @@
+import { ErrorHandler, Injectable, Injector } from "@angular/core";
+import { Store } from "@ngrx/store";
+import { StarkCoreApplicationState } from "../../../common/store";
+import { StarkUnhandledError } from "../actions";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "../../logging/services";
+
+@Injectable()
+export class StarkErrorHandling implements ErrorHandler {
+	private _starkLoggingService: StarkLoggingService;
+	private _applicationStore: Store<StarkCoreApplicationState>;
+
+	public constructor(private injector: Injector) {}
+
+	/**
+	 * Thie method will dispatch an error method, which the user can then handle
+	 * @param error the encountered error
+	 */
+	public handleError(error: any): void {
+		this.starkLoggingService.error("StarkErrorHandling: an error has occurred : ", error);
```

Comment:
This logging should say "StarkErrorHandler: ..." instead of ""StarkErrorHandling: ...."